packages/embark-swarm/src/storageProcessesLauncher.js

```diff
@@ -0,0 +1,180 @@
+import {__} from 'embark-i18n';
```

Comment:
I think this file lives in `embark-storage`. I think it can be used in both swarm and IPFS

Reply:
Like the blockchain plugins, this is something that iuri didn't want to abstract out. I am very keen to discuss this as part of our ongoing brainstorm about creating a blockchain plugin API.

Reply:
Agree